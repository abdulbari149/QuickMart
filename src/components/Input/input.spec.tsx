import setupComponentSnapshotTests from 'storybook/utils/setup-component-snapshot-tests';

import inputConfig from './input.config';

setupComponentSnapshotTests(inputConfig);
