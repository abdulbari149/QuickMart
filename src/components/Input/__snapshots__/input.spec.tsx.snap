// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given - Input component and props are: {"InputLeftElement":null,"InputRightElement":null,"label":"Email Address"} should render correctly and match snapshot 1`] = `
<View
  style={
    [
      {
        "alignItems": "center",
        "backgroundColor": "#FFFFFF",
        "borderColor": "#CCCCCC",
        "borderRadius": 24,
        "borderWidth": 1,
        "flexDirection": "row",
        "marginVertical": 14,
        "paddingHorizontal": 5,
        "paddingVertical": 27,
        "width": "100%",
      },
      {},
    ]
  }
>
  <View
    style={
      [
        {
          "flex": 1,
          "flexDirection": "column",
          "gap": 2,
          "paddingHorizontal": 34,
        },
        {},
      ]
    }
  >
    <Text
      style={
        [
          {
            "color": "#000000",
            "fontSize": 19,
            "fontWeight": "normal",
            "letterSpacing": 0.3,
          },
          {
            "color": "#0A4E8Eaa",
            "fontSize": 18,
            "paddingHorizontal": 1,
          },
          {},
        ]
      }
    >
      Email Address
    </Text>
    <TextInput
      accessibilityLabel="input field"
      autoCapitalize="none"
      autoCorrect={false}
      placeholderTextColor="#CCCCCC"
      style={
        [
          {
            "color": "#000000",
            "fontSize": 24,
          },
          {},
        ]
      }
    />
  </View>
</View>
`;

exports[`Given - Input component and props are: {"InputLeftElement":null,"InputRightElement":null} should render correctly and match snapshot 1`] = `
<View
  style={
    [
      {
        "alignItems": "center",
        "backgroundColor": "#FFFFFF",
        "borderColor": "#CCCCCC",
        "borderRadius": 24,
        "borderWidth": 1,
        "flexDirection": "row",
        "marginVertical": 14,
        "paddingHorizontal": 5,
        "paddingVertical": 27,
        "width": "100%",
      },
      {},
    ]
  }
>
  <View
    style={
      [
        {
          "flex": 1,
          "flexDirection": "column",
          "gap": 2,
          "paddingHorizontal": 34,
        },
        {},
      ]
    }
  >
    <TextInput
      accessibilityLabel="input field"
      autoCapitalize="none"
      autoCorrect={false}
      placeholderTextColor="#CCCCCC"
      style={
        [
          {
            "color": "#000000",
            "fontSize": 24,
          },
          {},
        ]
      }
    />
  </View>
</View>
`;

exports[`Given - Input component and props are: {"placeholder":"Enter your email","InputLeftElement":null,"InputRightElement":null,"label":"Email Address"} should render correctly and match snapshot 1`] = `
<View
  style={
    [
      {
        "alignItems": "center",
        "backgroundColor": "#FFFFFF",
        "borderColor": "#CCCCCC",
        "borderRadius": 24,
        "borderWidth": 1,
        "flexDirection": "row",
        "marginVertical": 14,
        "paddingHorizontal": 5,
        "paddingVertical": 27,
        "width": "100%",
      },
      {},
    ]
  }
>
  <View
    style={
      [
        {
          "flex": 1,
          "flexDirection": "column",
          "gap": 2,
          "paddingHorizontal": 34,
        },
        {},
      ]
    }
  >
    <Text
      style={
        [
          {
            "color": "#000000",
            "fontSize": 19,
            "fontWeight": "normal",
            "letterSpacing": 0.3,
          },
          {
            "color": "#0A4E8Eaa",
            "fontSize": 18,
            "paddingHorizontal": 1,
          },
          {},
        ]
      }
    >
      Email Address
    </Text>
    <TextInput
      accessibilityLabel="input field"
      autoCapitalize="none"
      autoCorrect={false}
      placeholder="Enter your email"
      placeholderTextColor="#CCCCCC"
      style={
        [
          {
            "color": "#000000",
            "fontSize": 24,
          },
          {},
        ]
      }
    />
  </View>
</View>
`;

exports[`Given - Input component and props are: {"placeholder":"Enter your email","InputLeftElement":null,"InputRightElement":null} should render correctly and match snapshot 1`] = `
<View
  style={
    [
      {
        "alignItems": "center",
        "backgroundColor": "#FFFFFF",
        "borderColor": "#CCCCCC",
        "borderRadius": 24,
        "borderWidth": 1,
        "flexDirection": "row",
        "marginVertical": 14,
        "paddingHorizontal": 5,
        "paddingVertical": 27,
        "width": "100%",
      },
      {},
    ]
  }
>
  <View
    style={
      [
        {
          "flex": 1,
          "flexDirection": "column",
          "gap": 2,
          "paddingHorizontal": 34,
        },
        {},
      ]
    }
  >
    <TextInput
      accessibilityLabel="input field"
      autoCapitalize="none"
      autoCorrect={false}
      placeholder="Enter your email"
      placeholderTextColor="#CCCCCC"
      style={
        [
          {
            "color": "#000000",
            "fontSize": 24,
          },
          {},
        ]
      }
    />
  </View>
</View>
`;
